documentation/experimental_features.md

```diff
@@ -366,6 +367,82 @@ in the result array, the plan throws a `PlanFailure` and includes the entire res
 `PlanFailure` and continues to execute the plan. If you've provided `_catch_errors => true`
 to `wait()`, Bolt returns any errors raised and the plan continues to execute.
 
+#### Viewing failing results in parallel blocks
+
+When an action fails in a parallel block, such as a task returning an error, Bolt
+does not print the error result to the console. For example, the following plan
+runs a non-existent command on targets in parallel:
+
+```puppet
+plan example (
+  TargetSpec $targets
+) {
+  $_targets = get_targets($targets)
+
+  $results = parallelize($_targets) |$target| {
+    run_command('badcommand', $target)
+  }
+
+  return $results
+}
+```
+
+The error result from each target the command is run is not printed to the console,
+making it difficult to know why the command failed:
```

Comment:
```suggestion
Bolt does not print the error result from the targets the command is run on,
making it difficult to know why the command failed:
```

Reply:
I think "to the console" is clear from the context here, so we can remove it to shorten this sentence a bit